.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
it's in review already: https://github.com/quic/efficient-transformers/pull/9/files

Reply:
Okay, I am removing. Actualy, I was trying to remove those cache files